cli/req.go

```diff
@@ -40,7 +41,11 @@ func (n Nexus3) request(url string) ([]byte, string, error) {
 }
 
 func (n Nexus3) response(req *http.Request) ([]byte, string, error) {
-	resp, err := http.DefaultClient.Do(req)
+	retryClient := retryablehttp.NewClient()
+	retryClient.RetryMax = 5
+	retryClient.Logger = nil
```

Comment:
I used https://github.com/hashicorp/go-retryablehttp here, it's nice, only one thing I want to highlight: by default it has debug logging. Even if I set Logger to logrus, it just wraps DEBUG messages inside logrus format and we still have a lot of noise. 

Disabling logger as I have disabled it looks fine, the only one thing - it's not reporting original error, only the last one which comes from `retryClient`.
E.g. original message: 
```
FATA[0279] ResponseCode: '500' and Message '500 javax.servlet.ServletException: org.sonatype.nexus.repository.storage.RetryDeniedException: Exceeded retry limit for URL: https://nexus-dev/repository/thirdparty/com/oracle/oim/wlthint3client/10.3.6.0/wlthint3client-10.3.6.0.pom 

```

How it looks after this change:

```
 12 / 4442 [>-------------------------------------------------------------------------------------------------------------------------------------------------]   0.27%
FATA[1160] Get https://nexus-dev/repository/thirdparty/com/oracle/oim/wlthint3client/10.3.6.0/wlthint3client-10.3.6.0.pom: GET https://nexus-dev.va/repository/thirdparty/com/oracle/oim/wlthint3client/10.3.6.0/wlthint3client-10.3.6.0.pom giving up after 6 attempts 
```

I tried to configure Logger to work with logrus, but no success here, I give up :)